src/aat/resources/features/F-127 - Even Enabling Condition/S-127.2.td.json

```diff
@@ -0,0 +1,49 @@
+{
+	"_guid_": "S-127.2",
+  "_extends_": "F-127_Get_Case_View_Base",
+  "title": "Get Case event view to validate the events returned in the response when event enabling conditions are valid",
```

Comment:
Should it be "are not valid" at the end? S-127.2 is for a non matching condition.